test/sharness/t0250-files-api.sh

```diff
@@ -316,13 +316,26 @@ test_files_api() {
 		verify_dir_contents /cats file1 ipfs this
 	'
 
+	test_expect_success "write 'no-flush' succeeds" '
+		echo "testing" | ipfs files write -f -e /cats/walrus
+	'
+
+	test_expect_success "changes bubbled up to root on inspection" '
+		ipfs files stat / | head -n1 > root_hash
```

Comment:
actually, can we test: 
- that they didn't bubble up beforehand?
- that the dag objects themselves are written and pinned (for safety)